uwsgitop

```diff
@@ -268,7 +268,7 @@ def main():
                     wrunt = str(wrunt)
 
                 if worker['last_spawn']:
-                    wlastspawn = time.strftime("%H:%M:%S", time.localtime(worker['last_spawn']))
+                    wlastspawn = time.strftime("%Y-%m-%dT%H:%M:%S", time.localtime(worker['last_spawn']))
```

Comment:
What about something like ps aux is doing? e.g. mar27 for tasks spawned yesterday.

Reply:
I think I prefer the full timestamp, as it's more precise and doesn't cost that much screen width... but I don't feel that strongly about it, so if you prefer the `ps aux` style I can see about implementing that. It would likely be much more code.

Reply:
Sorry, missed you were waiting for my feedback. Yes i think the logic should be:
- delta > 1 year, show year, e.g. 2019
- delta > 24h, show mar27
- show time